### ISSUE TITLE
ImageNet model experiment

### ISSUE THREAD

**Issue by JacopoDapueto** [Horizon2333/imagenet-autoencoder#7]
Hello!
thanks for sharing the model weights on Imagenet!
However, what are the hyperparameters used? I mean learning rate, weight decay, number of epochs, data augmentation, etc.

Thank you so much!

**Comment by Horizon2333** [comment]
Hello. 
Thank you for your interest in this project. 
As for the problem of hyperparameters, I'm sorry that many details have not been available since I have finished this project for more than two years. But some of the information I can provide is that in my memory, there is no significant difference between the networks trained on ImageNet and those trained on other data sets, otherwise I would have been impressed by the relevant parameter adjustments.
